dusty/systems/docker/compose.py

```diff
@@ -18,31 +18,33 @@ def _write_composefile(compose_config, compose_file_location):
     with open(compose_file_location, 'w') as f:
         f.write(yaml.dump(compose_config, default_flow_style=False))
 
-def _compose_up(compose_file_location, project_name, recreate_containers=True):
-    logging.info('Running docker-compose up')
+def _compose_base_command(core_command, compose_file_location, project_name=None):
+    logging.info('Running docker-compose {}'.format(core_command))
     command = ['docker-compose']
     if compose_file_location is not None:
```

Comment:
are you talking about the compose_file_location here? Do you mean I should remove the check around if it not None?